app/models/manageiq/providers/amazon/agent_coordinator.rb

```diff
@@ -59,7 +59,6 @@ def startup_agent
     agent_ids.empty? ? deploy_agent : activate_agents
   rescue => err
     _log.error("No agent is set up to process requests: #{err.message}")
-    _log.error(err.backtrace.join("\n"))
```

Comment:
Should we remove this backtrace, or just change it to `_log.error`?

Reply:
Most time this backtrace is duplicated with https://github.com/hsong-rh/manageiq-providers-amazon/blob/530f58b7bf0df538fe99859b0dd71cf6bf973efa/app/models/manageiq/providers/amazon/agent_coordinator.rb#L208

Reply:
OK.